backends/tc/introspection.h

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright (C) 2023 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions
+and limitations under the License.
+*/
+
+#ifndef BACKENDS_TC_INTROSPECTION_H_
+#define BACKENDS_TC_INTROSPECTION_H_
+
+#include "frontends/p4/parseAnnotations.h"
+#include "frontends/p4/parserCallGraph.h"
+#include "ir/ir.h"
+#include "lib/json.h"
+#include "lib/nullstream.h"
+#include "options.h"
+
+namespace TC {
+
+struct introspectionInfo {
```

Comment:
As @fruffy said, we usually use capitalized names for types.

Reply:
Updated with Capitalized names